src/implementations/react/src/elements/components/GlobalNav/GlobalNav.js

```diff
@@ -105,8 +107,8 @@ class GlobalNav extends Component {
           ? <ProjectAccountSwitcher
               accounts={this.props.accounts}
               projects={this.props.projects}
-              accountSectionTitle={this.props.accountSectionTitle}
-              projectSectionTitle={this.props.projectSectionTitle}
+              accountTitle={this.props.accountTitle}
+              projectTitle={this.props.projectTitle}
```

Comment:
These may belong on `this.props.topnav` OR we may want to rethink nesting props like that.